cluster/deploy.sh

```diff
@@ -49,6 +37,8 @@ elif [[ $TARGET =~ .*-release ]] || [[ $TARGET =~ windows.* ]]; then
     done
 fi
 
+_kubectl create -f ${MANIFESTS_OUT_DIR}/testing -R $i
```

Comment:
test infrastructure needs to be deployed before the kubevirt manifests.... the test manifests is where the kubevirt-config lives for our tests, and that has to be set before installing kubevirt